exercises/sublist/.meta/template.j2

```diff
@@ -0,0 +1,24 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{% macro test_case(case, translate) -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        self.assertEqual({{ case["property"] }}({{ case["input"]["listOne"] }}, {{ case["input"]["listTwo"] }}),
+                         {{ translate[case["expected"]] }})
```

Comment:
The `translate` dict is clever, but in this case you can use a builtin [filter] to accomplish this with lest boilerplate:

```suggestion
                         {{ case["expected"] | upper }})
```

[filter]: https://jinja.palletsprojects.com/en/2.10.x/templates/#filters